rest_auth/registration/views.py

```diff
@@ -91,31 +95,96 @@ def post(self, request, *args, **kwargs):
         return Response({'detail': _('ok')}, status=status.HTTP_200_OK)
 
 
-class SocialLoginView(LoginView):
-    """
-    class used for social authentications
-    example usage for facebook with access_token
-    -------------
-    from allauth.socialaccount.providers.facebook.views import FacebookOAuth2Adapter
+if 'allauth.socialaccount' in settings.INSTALLED_APPS:
```

Comment:
I moved the `SocialLoginView` definition into the conditional `if` block where the imports reside, I think this cleans up the code slightly.